src/main/resources/python/rectangle.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, text, target=None, height=0.75, width=1.5, color=BLUE, text_c
         self.pointer.next_to(self.shape, TOP, 0.01)
         if target:
             self.owner = target
-            self.all.scale(max(target.empty.submobjects[1].get_height() / self.shape.get_height(), target.empty.get_width() / self.shape.get_width()))
+            self.all.scale(max(target.empty.get_height() / self.shape.get_height(), target.empty.get_width() / self.shape.get_width()))
```

Comment:
Everything looks good to me! I think we have to change this a bit if we want to use the same Init_Structure class for linked list but we can do that when we do linked list.